cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/servlet/parameters/ParameterUtils.java

```diff
@@ -111,6 +112,7 @@
   public static final String POPULATE_DISK_INFO_PARAM = "populate_disk_info";
   public static final String BROKER_ID_AND_LOGDIRS_PARAM = "brokerid_and_logdirs";
   public static final String REPLICATION_FACTOR_PARAM = "replication_factor";
+  public static final String TOPIC_BY_REPLICATION_FACTOR_PARAM = "topic_by_replication_factor";
```

Comment:
I'm a little fuzzy on the desired API currently.

Is `topic_by_replication_factor` intended to signify a mapping from RF (positive integers) to topic regex?

Is the `topic_by_replication_factor` codepath expected to be accessible only from valid JSON submitted in the body of the request (this appears to be what the subsequent `topicPatternByReplicationFactorFromBody` method implements) , or would it potentially also be a new parameter that could be sent to `cruise-control` (I don't see this anywhere, nor do I think this would be a good idea, I just see `TOPIC_BY_REPLICATION_FACTOR_PARAM` next to other existing parameter Strings, and I thought I would ask)

Reply:
After reading the updates to `TopicConfigurationParameters.java`, most of my questions are answered.

I do still have a question for whether `topic_by_replication_factor` is ever potentially a parameter in the URI, even though it's expected to be a key in a JSON body sent to `cruise-control`

Reply:
`topic_by_replication_factor` will be only used in request body, `topic_configuration` endpoint's parameter list does not change in this patch.

Reply:
Thanks Kun, all of my questions here have been addressed